numericalEFT/MCIntegration.jl
# Thread

1. Test thread compatibility
2. Improve API to (1) support julia do-syntax (see readme). (2) make the functions arguments dictionary.
3. Remove some allocations. The code is 30% faster.
4. Add ProgressMeter support. Now print=0 or print>0 will show the progress of integration.
